core/src/test/scala/integration/kafka/server/KRaftClusterTest.scala

```diff
@@ -832,4 +833,31 @@ class KRaftClusterTest {
     }
   }
 
+
+  @Test
+  def testUpdateMetadataVersion(): Unit = {
```

Comment:
This test case will fail without this change, can you take a look at this PR @jsancio , I wish this can be merged into 3.3.

Reply:
Thanks. I'll mark as a blocker for now but it would be good for @mumrah or @cmccabe to comment and review.